plugins/infinispan70/src/main/java/org/radargun/service/InfinispanClientListeners.java

```diff
@@ -92,8 +92,8 @@ public void removeExpiredListener(String cacheName, ExpiredListener listener, bo
    public void removeListener(String cacheName, boolean sync) {
       GenericClientListener listener = getListenerOrThrow(cacheName, sync);
       if (listener.isEmpty()) {
-         service.getRemoteManager(false).getCache(cacheName).removeClientListener(listener);
-         service.getRemoteManager(true).getCache(cacheName).removeClientListener(listener);
+         service.getRemoteManager(false).getCache(cacheName == null ? "" : cacheName).removeClientListener(listener);
```

Comment:
Does it mean that RCM.getCache("") should return a default cache? I went through RCM class and documentation doesn't mention anything about this. I'd double check whether the method doesn't create cache with empty name. What about using just getCache() when 'cacheName' param is empty?